RELEASE_CHANGELOG.md

```diff
@@ -0,0 +1,48 @@
+## [v1.8.0-rc8](https://github.com/provenance-io/provenance/releases/tag/v1.8.0-rc8) - 2022-02-16
```

Comment:
I don't think we want to commit this file to the `release/v1.8.x` branch...

Reply:
no... release_changelog.md should not exist in a `.x` release branch

Reply:
sounds good, commit dropped